src/text.rs

```diff
@@ -1,31 +1,10 @@
 //! Reading lines as events from a text input.
 
-use crate::Error;
+use crate::{BareEvent, Error};
 use std::io::{BufRead, BufReader, Read};
 use std::mem;
 
-/// A line in a text input.
-#[derive(Debug)]
-pub struct Event {
-    pub raw: Vec<u8>,
-    pub line_no: u64,
-}
-
-impl crate::Event for Event {
-    type Ack = u64;
-
-    fn raw(&self) -> &[u8] {
-        self.raw.as_slice()
-    }
-
-    fn time(&self) -> u64 {
-        self.line_no
-    }
-
-    fn ack(&self) -> Self::Ack {
-        self.line_no
-    }
-}
+type Event = BareEvent;
```

Comment:
This needs to be `pub`; otherwise a user of this crate cannot use `text::Event`.